changelog/12668.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
I'm not sure if this should be marked as a feature or an improvement. Leaning towards the latter.